packages/manager/modules/bm-server-components/src/consumption-tile/component.js

```diff
@@ -0,0 +1,12 @@
+import controller from './controller';
+import template from './template.html';
+
+export default {
+  bindings: {
+    server: '<',
+    trafficInformations: '<',
```

Comment:
**niptick** : information is always in singular

```suggestion
    trafficInformation : '<',
```

Reply:
I applied for the complete refactor